### DISCUSSION TITLE
uYouEnhanced Announcements

### DISCUSSION THREAD

**Discussion by arichornlover** [arichornlover/uYouEnhanced#2] [Announcements]
# a channel where announcements are made.

**Comment by arichornlover** [comment]
the release **v18.31.3-3.0.1** was supposed to be **v18.32.2-3.0.1** but I sadly did not have the latest ipa and wasn’t able to obtain a new one this week so I did my best. I’m sorry if this frustrates some of y’all 😥

**Reply by yodaluca23** [reply]
Yea, it's not on ArmConverter yet either...

**Reply by arichornlover** [reply]
and that was the site I was using to obtain those ipas 😞

**Comment by arichornlover** [comment]
# New SDK Changes

uYouEnhanced will no longer be using the iOS 15.5 SDK. It’s now gonna start using the iOS 16.2 SDK which is more up-to-date and is required for the tweak.

**Reply by ego-lay-atman-bay** [reply]
Does this mean that me, stuck on ios 15.3.1 will not be able to use uYouEnhanced?

**Reply by arichornlover** [reply]
No, you can still use it, all I did was update the SDK, and the uYouEnhanced tweak still works on all compatible iOS/iPadOS Versions.

**Reply by arichornlover** [reply]
Update: it uses the iOS 16.4 SDK now.

**Reply by aricloverEXTRA** [reply]
Update Again: it uses iOS 17.5 SDK now. Probably Forgot to mention it.

**Comment by arichornlover** [comment]
As of January 19th 2024, @LillieH1000 has archived the YouTube Reborn Tweak. However I have a forked version! https://github.com/arichornlover/YouTube-Reborn-v5

On my version of YouTube Reborn, you’ll get a Better YouTube Reborn Menu and localization Support that weren’t in LillieH1000’s Version. Honestly the YouTube Reborn tweak is a good one for maintainability! if you want all the tweaks from uYouEnhanced you can install YouTubeRebornPlus! https://github.com/arichornlover/YouTubeRebornPlus

**Comment by arichornlover** [comment]
Hey everyone, just a heads up about a website called https://uyouplus.com.

I strongly advise against downloading anything from this site. It's distributing IPAs from this repository without permission, and I'm unsure of the site's creator. Instead, I recommend creating your own uYouEnhanced IPAs with your own decrypted YouTube ipa using github actions. There’s probably a risk of malware with downloads from that site, so it's best to avoid downloading them. I do not endorse or support this site in any way.

**Comment by arichornlover** [comment]
## Announcement (04-15-24)
Hello y’all!
Google recently cracked down on YouTube third-party apps including uYouEnhanced due to copyright concerns. You can find more information on Google's enforcement of third-party apps on their official thread here: https://support.google.com/youtube/thread/269521462?hl=en

**Comment by arichornlover** [comment]
## Announcement (05-28-24)

Hello uYouEnhanced Members / Contributors
I have an unfortunate announcement regarding the YouTube Version’s `v19.21.2 or higher`
so Google has published a release called **v19.21.2** and what it changed basically removes `iOS 14 Compatibility`

so **v19.21.2** requires `iOS/iPadOS 15 or higher`


and no worries, to anyone being afraid of this! I will also do a final release of **v19.20.2** that still has iOS 14 compatibility on the *apps_legacy* version of the uYouEnhanced AltSource. https://github.com/arichornlover/arichornlover.github.io/blob/main/apps-legacy.json
hope this information is useful.

**Comment by aricloverEXTRA** [comment]
YouTube v19.41.3 introduces an updated MiniPlayer from what I know.
This happened to me when I updated from 19.40.4 to 19.41.3.

not that it matters or anything but I could make an option to revert the changes if anyone prefers the original MiniPlayer design.

**Reply by I-794** [reply]
I think it would be a nice addition for people who prefer the old mini player design

**Reply by aricloverEXTRA** [reply]
> I think it would be a nice addition for people who prefer the old mini player design

I think I can easily revert it too. This new MiniPlayer design can be found in the “A/B” settings of the uYouEnhanced/uYouPlus app.

**Comment by arichornlover** [comment]
YouTube v19.43.2 and higher introduces **NEW MATERIAL ICONS**
![IMG_2397](https://github.com/user-attachments/assets/b2a040c5-34ca-464d-a398-e5c208409099)
![IMG_2396](https://github.com/user-attachments/assets/a20cfd8d-8e23-4776-b452-83522dca4c75)

this change cannot be reversed (except for version sooofing). But they definitely look a lot better than those previous icons.

**Comment by aricloverEXTRA** [comment]
# 🧪Experimental Option in **uYouEnhanced 19.49.7-3.0.4** and higher.
![Subscribe!](https://github.com/user-attachments/assets/440f6a1f-6db8-4fbb-bf5c-f9865bffb9af)
```
redSubscribeButton_enabled: 1
```
Copy the text to clipboard and then press `Paste Settings` in uYouEnhanced Settings to activate it.

**What does it do?:** it changes the color of the Subscribe button from “white/black” back to **RED**. But this partially works unfortunately. So basically if you ever use the subscribe button, it’ll turn red.

**How do I turn it off?:** do the same thing like I said earlier except replace “1” with “0”, here is an example below.
```
redSubscribeButton_enabled: 0
```
